bears/coffee_script/CoffeeLintBear.py

```diff
@@ -22,11 +23,289 @@ class CoffeeLintBear:
     CAN_DETECT = {'Syntax', 'Formatting', 'Smell'}
 
     severity_map = {'warn': RESULT_SEVERITY.NORMAL,
-                    'error': RESULT_SEVERITY.MAJOR}
+                    'error': RESULT_SEVERITY.MAJOR,
+                    'ignore': RESULT_SEVERITY.INFO}
 
     @staticmethod
     def create_arguments(filename, file, config_file):
-        return '--reporter=raw', filename
+        return '--reporter=raw', '-f', config_file, filename
+
+    @staticmethod
+    def generate_config(filename, file,
+                        max_line_length: int=79,
```

Comment:
even if we change it to 79, i think it would be inconsistent to have it in this commit. rather leave it 80 and make a new PR where you change it globally to other bears too :)